includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2072,6 +2072,13 @@ protected function get_account_branding_secondary_color( $default_value = '' ):
 		return $default_value;
 	}
 
+	/**
+	 * Retrieves the country of the current account.
+	 */
+	protected function get_account_country(): string {
```

Comment:
`get_account_domestic_currency` + see [this method](https://github.com/Automattic/woocommerce-payments/blob/develop/includes/class-wc-payments-localization-service.php#L93). 

We shouldn't do the mapping on JS if we have working code on the PHP side.

Reply:
thanks @kalessil for the hint. I changed the approach here [a563fff](https://github.com/Automattic/woocommerce-payments/pull/6920/commits/a563fffd19cc550bbd93ed1f7e2cfd68267db547)